<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="yaml_path" default="/home/movel/"/>
  <node pkg="path_recall" type="path_load_segments_node" name="path_load" output="screen">
    <rosparam file="$(find path_recall)/config/path_recall.yaml"/>
    <rosparam file="$(env HOME)/.config/movel/config/path_recall/config/path_recall.yaml"/>
    <param name="yaml_path" value="$(arg yaml_path)" type="str" />
    <!--remap from="plan" to="move_base_simple/recorded_path"/-->
  </node>
</launch>
